phpunit.xml.dist

```diff
@@ -19,6 +19,8 @@
         <env name="SYMFONY_PHPUNIT_VERSION" value="9.6" />
         <env name="MAKER_SKIP_MERCURE_TEST" value="false"/>
         <env name="MAKER_SKIP_PANTHER_TEST" value="false" />
+<!--    Overrides process timeout when step debugging -->
+<!--    <env name="MAKER_PROCESS_TIMEOUT" value="null" /> -->
```

Comment:
Allows for debugging `Process::fromShellCommandLine()` which sets the process `$timeout` to 60 seconds